### ISSUE TITLE
Code block black on white

### ISSUE THREAD

**Issue by tdegeus** [jothepro/doxygen-awesome-css#1]
Thanks @jothepro for this nice theme ! It's brilliant that you made it and share it. And thanks for recommending it here : https://github.com/doxygen/doxygen/issues/8404 .

Before going to the title of this "issue", a question. I guess that the license is such that I can just copy the `css` file and include it in my repo (to keep things simple I'd rather steer around submodules however good they work, even though I would have to update myself). But, do you have a way in mind to credit your work?

Then, I do have some suggestions:

## Whitespace 

I feel that too lot of white space is 'wasted'. Consider the following example with a few long lines (generated on exactly the same browser window):

<img width="1890" alt="Screenshot 2021-03-19 at 19 36 13" src="https://user-images.githubusercontent.com/7030445/111827684-7b763000-88ea-11eb-9422-b5bd206d92d8.png">
<img width="1900" alt="Screenshot 2021-03-19 at 19 36 42" src="https://user-images.githubusercontent.com/7030445/111827696-7f09b700-88ea-11eb-8e4b-072ba90b2c74.png">

I think that this really make the docs less readable, as I find a key strength of the doxygen rendering (despite, I agree with you, its old looks) it getting in overview. Personally, I think that herein the doxygen rendering really has its merits for example over breathe. 

## Summary 

Personally, I feel that the summary of functions could be a bit brighter, it now almost disappears in the background:

<img width="949" alt="Screenshot 2021-03-19 at 19 23 08" src="https://user-images.githubusercontent.com/7030445/111826227-b7a89100-88e8-11eb-9113-a3514dccf31e.png">

## Code block

I have some code blocks, and those are shown black on white, which does not look super nice

<img width="908" alt="Screenshot 2021-03-19 at 19 25 32" src="https://user-images.githubusercontent.com/7030445/111826366-e6266c00-88e8-11eb-9ed6-51d7585cd559.png">

for which my docstring was 

```cpp
    /**
    Extract the unknown "dofval":

        dofval[iiu()]

    \param dofval input [#ndof]
    \return dofval_u input [#nnu]
    */
```

## Dependency graph

Here there is nothing to do I think. But the dependency graphs could look better ;)

<img width="246" alt="Screenshot 2021-03-19 at 19 28 31" src="https://user-images.githubusercontent.com/7030445/111826730-58974c00-88e9-11eb-9df7-2ac2dd4c137b.png">

**Comment by jothepro** [comment]
Hey Tom, 

thanks for your detailed feedback!

> do you have a way in mind to credit your work?

I've licensed the theme as MIT so you can do whatever you want with it! 

All I as a the theme author could do to add credits is to add the name of the theme with some css magic, but without it being a link I don't think its helpful:
<img width="332" alt="Bildschirmfoto 2021-03-19 um 19 56 28" src="https://user-images.githubusercontent.com/21294002/111829642-3bfd1300-88ed-11eb-92a3-34b3fcdc5d31.png">

If you want to give me credits by adding a link in the footer-html, you're welcome! 😊 


> I feel that too lot of white space is 'wasted'.

You can configure how wide the content should be by overwriting the css-variable `--content-maxwidth`. The default value is 900px. If you want to restore the default behaviour of doxygen (fill all available space with content), you should set it to `auto`:

```css
:root {
  --content-maxwidth: auto;
}
```

Documentation of variables is WIP, I may add this information to the docs in the future.

> Personally, I feel that the summary of functions could be a bit brighter
> ...
> I have some code blocks, and those are shown black on white,

Good point. I will fix these things. Stay tuned!

> Dependency graph
> Here there is nothing to do I think. But the dependency graphs could look better ;)

I think there can be done something! Simply inverting the colors of the image in dark mode may help a lot. I'll check that out.

**Comment by jothepro** [comment]
@tdegeus I think I have fixed the problems you described in [v1.1.0](https://github.com/jothepro/doxygen-awesome-css/releases/tag/v1.1.0). Please close this issue if you can confirm that. 👏

**Comment by tdegeus** [comment]
@jothepro Looks amazing!!! Thanks!
